atom.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <feed xmlns="https://www.w3.org/2005/Atom">
   <title>{{ config.title }}</title>
-  {% if icon %}<icon>{{ icon }}</icon>{% endif %}
+  {% if icon %}
+  <image>
+    <url>{{ icon }}</url>
+    <title>{{ config.title }}</title>
+    <link>{{ url }}</link>
+  </image>
+  {% endif %}
```

Comment:
please revert this change, rss2 spec doesn't apply to atom.

Reply:
Oh.. OK I'll revert that commit